airbyte-integrations/connectors/source-pinterest/acceptance-test-config.yml

```diff
@@ -1,72 +1,68 @@
-# See [Source Acceptance Tests](https://docs.airbyte.com/connector-development/testing-connectors/source-acceptance-tests-reference)
-# for more information about how to configure these tests
-connector_image: airbyte/source-pinterest:dev
 acceptance_tests:
-  spec:
+  basic_read:
     tests:
-      # TODO: remove backward compatibility checks once updated to version `>0.1.3`
-      # because for OAuth2.0 implementation the specs are different
-      - spec_path: "source_pinterest/spec.json"
-        backward_compatibility_tests_config:
-          disable_for_version: "0.1.2"
+    - config_path: secrets/config.json
+      empty_streams:
+      - bypass_reason: The stream could return 0 records, because of low rate-limits
+        name: ad_account_analytics
```

Comment:
Can you explain this a little more please?  Rather than sleep to wait for more rate-limit, the source will give up and return 0 records?